src/components/SummaryPage.tsx

```diff
@@ -77,7 +224,7 @@ const SummaryPage = () => {
           Database summary
         </Typography>
         <Typography variant="body1" color={gray500}>
-          Last updated on September 15, 2023
+          Last updated on May 15, 2024
```

Comment:
Note (non-blocking): This update strategy seems very limiting in the sense that would require code changes everytime we want to update the content of this component. We will need to rethink this when we have a more concrete picture and more time